core/src/main/resources/defaults/blueprints/7.2.15/cdp-flow-management-small.bp

```diff
@@ -161,6 +161,10 @@
                 "name": "java.arg.2",
                 "value": "-Xms8g"
               },
+              {
+                "name": "java.arg.7",
```

Comment:
are we sure that the current 7.2.15 image now using the nifi version which contains this ?

Reply:
@doktoric current 7.2.15 image contains CFM-2.2.4.0-98 version is too old (we already have CFM-2.2.4.0-141). Also CFM-2.2.4.0 is related to CDP 7.2.14. For CDP 7.2.15 we will use new CFM-2.2.5.0 version. Current change is required for CFM-2.2.4.0 and newer versions, so it should be committed.

Reply:
@doktoric ok, will wait.

Reply:
@doktoric now 7.2.15 images contain CFM-2.2.4.0-143 which has related changes, so current change could be successfully committed and will not fail.